docs/SnapPacksApi.md

```diff
@@ -0,0 +1,317 @@
+
+
+## snap_pack_retrieve
+
+> <SnapPack> snap_pack_retrieve(snp_id)
+
+get
+
+Retrieves the details of an existing snap_pack. You need only supply the unique snap_pack identifier that was returned upon snap_pack creation.
+
+### Examples
+
+```ruby
+require 'time'
+require 'lob'
+# setup authorization
+Lob.configure do |config|
+  # Configure HTTP basic authorization: basicAuth
+  config.username = 'YOUR USERNAME'
+  config.password = 'YOUR PASSWORD'
+end
+
+api_instance = Lob::SnapPacksApi.new
+snp_id = 'snp_id_example' # String | id of the snap_pack
+
+begin
+  # get
+  result = api_instance.get(snp_id)
+  p result
+rescue Lob::ApiError => e
+  puts "Error when calling SnapPacksApi->get: #{e}"
+end
+```
+
+#### Using the snap_pack_retrieve_with_http_info variant
+
+This returns an Array which contains the response data, status code and headers.
+
+> <Array(<SnapPack>, Integer, Hash)> snap_pack_retrieve_with_http_info(snp_id)
+
+```ruby
+begin
+  # get
+  data, status_code, headers = api_instance.snap_pack_retrieve_with_http_info(snp_id)
+  p status_code # => 2xx
+  p headers # => { ... }
+  p data # => <SnapPack>
+rescue Lob::ApiError => e
+  puts "Error when calling SnapPacksApi->snap_pack_retrieve_with_http_info: #{e}"
+end
+```
+
+### Parameters
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
+| **snp_id** | **String** | id of the snap_pack |  |
+
+### Return type
+
+[**SnapPack**](SnapPack.md)
+
+### Authorization
+
+[basicAuth](../README.md#basicAuth)
+
+### HTTP request headers
+
+- **Content-Type**: Not defined
+- **Accept**: application/json
+
+
+## snap_packs_list
+
+> <SnapPackList> snap_packs_list(opts)
+
+list
+
+Returns a list of your Snap Packs. The snap packs are returned sorted by creation date, with the most recently created snap packs appearing first.
+
+### Examples
+
+```ruby
+require 'time'
+require 'lob'
+# setup authorization
+Lob.configure do |config|
+  # Configure HTTP basic authorization: basicAuth
+  config.username = 'YOUR USERNAME'
+  config.password = 'YOUR PASSWORD'
+end
+
+api_instance = Lob::SnapPacksApi.new
+opts = {
+  limit: 56, # Integer | How many results to return.
+  before: 'before_example', # String | A reference to a list entry used for paginating to the previous set of entries. This field is pre-populated in the `previous_url` field in the return response. 
+  after: 'after_example', # String | A reference to a list entry used for paginating to the next set of entries. This field is pre-populated in the `next_url` field in the return response. 
+  include: ['inner_example'], # Array<String> | Request that the response include the total count by specifying `include[]=total_count`. 
+  date_created: { key: Time.now}, # Hash<String, Time> | Filter by date created.
+  metadata: { key: 'inner_example'}, # Hash<String, String> | Filter by metadata key-value pair`.
+  size: [Lob::SnapPackSize::N6X18_BIFOLD], # Array<SnapPackSize> | The Snap Pack sizes to be returned.
```

Comment:
We do not support this size with snap packs. Should we update such that we refer to the available sizes and not the self-mailer sizes?